Sources/UINotificationQueue.swift

```diff
@@ -41,13 +41,13 @@ internal final class UINotificationQueue {
     /// - Returns: The created notification request.
     @discardableResult func add(_ notification: UINotification, notificationViewType: UINotificationView.Type, dismissTrigger: UINotificationDismissTrigger? = nil, allowDuplicates: Bool = false) -> UINotificationRequest {
```

Comment:
Note that this project is not at all compliant with our current linter configurations. I've decided to not update this as part of this PR, as it would result in a lot of additional changes.

Reply:
Is there an issue tracking it?

Reply:
Totally fine I think :)

Reply:
I've created a ticket in JIRA: https://wetransfer.atlassian.net/browse/TMOB-4254